### PR TITLE
[PSDK-38] Address-Scoped Private Key Export

### PR DESCRIPTION
**Pull request by John-peterson-coinbase**
### What changed? Why?
- Add `#export` to `Address` to export private key as a hex string
- Fixes various linter errors using `make format`

### CODE REVIEW

File: lib/coinbase/user.rb [hunk 1]
```diff
@@ -81,7 +81,7 @@ def save_wallet(wallet, encrypt: false)
       iv = ''
       if encrypt
         shared_secret = store_encryption_key
-        cipher = OpenSSL::Cipher::AES256.new(:GCM).encrypt
+        cipher = OpenSSL::Cipher.new('aes-256-gcm').encrypt
```
Comment (yuga-cb) [inline lib/coinbase/user.rb:84]: Why change this?

File: lib/coinbase/address.rb [hunk 2]
```diff
@@ -156,6 +156,12 @@ def faucet
       end
     end
 
+    # Exports the Address's private key to a hex string.
+    # @return [String] The Address's private key as a hex String
+    def export
```
Comment (yuga-cb) [inline lib/coinbase/address.rb:161]: Add test

**Comment by yuga-cb** [comment]
Please run `make docs`

**Comment by John-peterson-coinbase** [comment]
> Please run `make docs`

Done!

**Review by yuga-cb — APPROVED** [review]

**Pull request merged**